ember/.eslintrc.js

```diff
@@ -1,5 +1,56 @@
 "use strict";
 
 module.exports = {
-  extends: ["@adfinis-sygroup/eslint-config/ember-app"],
+  root: true,
```

Comment:
Please use the shared config, like it was before, but with the updated package name.

```
module.exports = {
  extends: "@adfinis/eslint-config/ember-app",
};
```

Reply:
This is not resolved yet. Please do not duplicate the whole config. Everything underneath the `extends` option is not necessary since it's already preset in the preset. Please check [the stuff you use](https://github.com/adfinis/eslint-config/blob/main/ember-app.js).

Reply:
:ping_pong: